server/rogue/endpoint-posts.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const RogueEndpoint = require('./endpoint');
+
+class RogueEndpointPosts extends RogueEndpoint {
+  constructor(client) {
+    super(client);
+    this.endpoint = 'posts';
+  }
+  /**
+   * index - Retrieve all Posts
+   *
+   * @see https://github.com/DoSomething/rogue/blob/master/docs/endpoints/posts.md#retrieve-all-posts
+   * @param  {Object} query
+   * @return {Promise}
+   */
+  index(query) {
+    return this
+      .executeGet(this.endpoint, query)
+      .then(responseBody => responseBody);
+  }
+  /**
+   * get - Retrieve a single Post
+   *
+   * @see https://github.com/DoSomething/rogue/blob/master/docs/endpoints/posts.md#retrieve-a-specific-post
+   * @param  {string|number} id
+   * @param  {Object} query
+   * @return {Promise}
+   */
+  get(id, query) {
```

Comment:
I think this would be a good change to make. In our other Gateway modules (see [browser](https://github.com/DoSomething/gateway-js/blob/d98796220aee360c11e51ef8c0c6d4e0e30b2187/src/RestApiClient.js#L72-L89) and [PHP](https://github.com/DoSomething/gateway/blob/58c68e3746fc4fb93f08bbd711f0cd97cd978b8f/src/Common/RestApiClient.php#L78-L93)), we use `get()`, `post()`, etc. as shorthands for making plain requests.